packages/auto-label/src/auto-label.ts

```diff
@@ -265,7 +265,7 @@ handler.autoLabelOnPR = async function autoLabelOnPR(
       logger.info(
         `Language label added to PR #${pull_number} in ${owner}/${repo} is ${language_label}`
       );
-      new_labels.push(language_label);
+      new_labels.concat(language_label);
```

Comment:
This doesn't change either array.  I don't imagine that was intended because the return value containing the concatenated array is dropped.

I think this should be
```
new_leabels.push(...language_label);
```

Reply:
I think, since we're definitely changing getLabel to return a string array, this is safe to do. The concern is that, with the old behavior, we could potentially return a string, and then adding a spread operator would add each character as an array value. But since we're switching over to returning an array, this is safe to do!